### PR TITLE
Update spec file to newer version

### PR DESCRIPTION
**Pull request by jkonecny12**
Adapt spec file for daily builds on Fedora.

### CODE REVIEW

File: contrib/kakoune.spec [hunk 1]
```diff
@@ -83,6 +83,9 @@ popd
 
 
 %changelog
+* Thu Jan 16 2020 Jiri Knoceny <jkonecny@redhat.com> - 2020.08.04-1
```
Comment (caksoylar) [inline contrib/kakoune.spec:86]: Fully aware of the irony of correcting the author's name but thought I'd ask anyway, is the last name a typo?

Comment (jkonecny12) [reply contrib/kakoune.spec:86]: :D :+1: Good one!

Comment (jkonecny12) [reply contrib/kakoune.spec]: Even the date was wrong. Should be fine now.

**Comment by jkonecny12** [comment]
Ping, could you please merge this?

**Pull request merged**